### PR TITLE
Testing pull request

### PR DESCRIPTION
**Pull request by efgramsbergen**
this only affects README.md

@tullemich, @mosart

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+
+**Favorite book at the moment:** Getallen zijn je beste vrienden (numbers are your best friends).
+
+**Favorite cities:** Rome, New York (but this list may change depending on my mood. Today I think Copenhagen is quite nice but things may change when it is raining tomorrow)
+
+**Favorite formulas:** Maxwell's equations. I will wear them on a T-shirt tomorrow for everyone to admire.
+
+![Maxwell's equations)](http://www.maxwells-equations.com/maxwells-equations.gif)
+
+**Favorite cartoon character:** Spongebob. Really? Yes, really.
```
Comment (mosart) [inline README.md:16]: you should put an image og sponge bob. for example this one

**Comment by tullemich** [comment]
Looks fine, but maybe I too big equation image on a web site. On the other hand if you find it REALLY important the proportions might be right! :end:

File: README.md (hunk 1 above)
Comment (efgramsbergen) [inline README.md:16]: That is a really great idea!

Comment (mosart) [inline README.md:16]: if you edit your file and commit on this line, maybe this comment will be hidden. can you try ?

**Comment by efgramsbergen** [comment]
Image can not be resized as far as I know, so I will ignor your highly esteemed remark.

**Pull request merged**